packages/devextreme/js/__internal/ui/chat/chat_message_group.ts

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import $ from '@js/core/renderer';
+import type { Message } from '@js/ui/chat';
+import type { WidgetOptions } from '@js/ui/widget/ui.widget';
+
+import Widget from '../widget';
+import Avatar from './chat_avatar';
+import MessageBubble from './chat_message_bubble';
+
+const CHAT_MESSAGE_GROUP_CLASS = 'dx-chat-message-group';
+const CHAT_MESSAGE_GROUP_ALIGNMENT_START_CLASS = 'dx-chat-message-group-alignment-start';
+const CHAT_MESSAGE_GROUP_ALIGNMENT_END_CLASS = 'dx-chat-message-group-alignment-end';
+const CHAT_MESSAGE_GROUP_INFORMATION_CLASS = 'dx-chat-message-group-information';
+const CHAT_MESSAGE_TIME_CLASS = 'dx-chat-message-time';
+const CHAT_MESSAGE_NAME_CLASS = 'dx-chat-message-name';
+const CHAT_MESSAGE_BUBBLE_FIRST_CLASS = 'dx-chat-message-bubble-first';
+const CHAT_MESSAGE_BUBBLE_LAST_CLASS = 'dx-chat-message-bubble-last';
+
+export interface MessageGroupOptions extends WidgetOptions<MessageGroup> {
+  messages: Message[];
+  alignment: 'start' | 'end';
+}
+
+class MessageGroup extends Widget<MessageGroupOptions> {
+  _avatar?: Avatar;
+
+  _getDefaultOptions(): MessageGroupOptions {
+    return {
+      ...super._getDefaultOptions(),
+      messages: [],
+      alignment: 'start',
+    };
+  }
+
+  _getAlignmentClass(): string {
+    const { alignment } = this.option();
+
+    const alignmentClass = alignment === 'start'
+      ? CHAT_MESSAGE_GROUP_ALIGNMENT_START_CLASS
```

Comment:
I would prefer to start with a single class for elements that need to be aligned to the opposite edge. In this case, we would simply override the default styles.

Reply:
It is always more convenient to work with two classes than with one. At least, it makes the whole code more readable. I don't see any disadvantages.

Reply:
Well, I will trust you.